tests/cases/stable/py/intercontract/alloc_budget_exhausted.py

```
@@ -0,0 +1,16 @@
+# { "Depends": "py-genlayer:test" }
+import genlayer as gl
+from genlayer.types import *
```

Ruff (0.15.15)

[error] `from genlayer.types import *` used; unable to detect undefined names

(F403)